Week1/Day2/ExerciseXP/code_XP.py

```diff
@@ -0,0 +1,114 @@
+#Exercise 1 : Set
+my_fav_numbers = set([2,3,5,7,8,10,12,17,21,25])
+my_fav_numbers.update([27,30])
+#Remove the last number --- We can do it only if we know the value of rhe "last" item, because set does not have indexes
+# For ex:my_fav_numbers.remove(30)
+
+friend_fav_numbers = set([3, 11, 18])
+# Concatenate my_fav_numbers and friend_fav_numbers to a new variable called our_fav_numbers
+our_fav_numbers = my_fav_numbers|friend_fav_numbers
```

Comment:
amazing! youre the only one that used this operator

Reply:
Thank you!